ql/src/test/org/apache/hadoop/hive/ql/TestTxnNoBuckets.java

```diff
@@ -848,7 +848,7 @@ public void testCompactStatsGather() throws Exception {
             .getParameters();
     Assert.assertEquals("The number of files is differing from the expected", "1", parameters.get("numFiles"));
     Assert.assertEquals("The number of rows is differing from the expected", "4", parameters.get("numRows"));
-    Assert.assertEquals("The total table size is differing from the expected", "704", parameters.get("totalSize"));
+    Assert.assertEquals("The total table size is differing from the expected", "705", parameters.get("totalSize"));
```

Comment:
I saw you have removed `totalSize  `in the UT of `TestCompactor.java`, so should we remove this line as well?

Reply:
I think it would be better to handle this in a separate ticket as refactoring. Would this be okay so we can close this ticket?

Reply:
sure

Reply:
we're removing some totalSize assertions but not others, let's simply keep it consistent, choose 1 option now from the below ones:
1. remove all occurrences that have been changed here by the patch, so it's visible now
2. remove only from qtests
3. remove all occurrences from the code -> time-consuming, I'm not recommending this
4. don't remove any totalSize assertions

I believe only 1) makes sense

Reply:
I just unresolved this conversation, it's time to finally address this whole totalSize somehow
option 5) would be to create another ticket to remove that stuff, merge ASAP, and then we can see how clear is this upgrade alone

Reply:
Option 5 means we remove only the totalSize checks which are in this ticket and not others, which would be a little bit weird, because it is hard to address which to eliminate outside this ticket. How can be the other ticket reviewed without this? So I would say create a ticket with clear description which to delete: a) from Qtests, b) Qtest and unit tests. Or create 2 new ticket, one for Qtests only, one for unit tests only. Since the build env is really unstable I had to run many times this build to finally succeed.

Reply:
okay, makes sense, so we can agree to remove totalSize checks from every places that were affected by this ORC upgrade and do the rest in subsequent patches, like [HIVE-27791](https://issues.apache.org/jira/browse/HIVE-27791)